charts/virtual-kubelet-0.1.1-rbac/values.yaml

```diff
@@ -0,0 +1,25 @@
+image:
+  repository: microsoft/virtual-kubelet
+  tag: latest
+  pullPolicy: Always
+env:
+  azureClientId: 
+  azureClientKey: 
+  azureTenantId: 
+  azureSubscriptionId: 
+  aciResourceGroup: 
+  aciRegion: 
+  nodeName: 
+  nodeTaint: 
+  nodeOsType: 
+  apiserverCert: 
+  apiserverKey: 
+
+## Install Default RBAC roles and bindings
+rbac:
+  install: false
+  serviceAccountName: default
```

Comment:
looks like not used, and I don't see a strong requirement to specify it when running the helm chart.

Reply:
Now referenced